permissions/plugin-configuration-as-code-groovy.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: "configuration-as-code-groovy"
+github: "jenkinsci/configuration-as-code-groovy-plugin"
+paths:
+- "org/jenkins-ci/plugins/configuration-as-code-groovy"
```

Comment:
Is this path needed? Your groupId is "io.jenkins"

Reply:
Please check now. I do not get this "paths" part